### ISSUE TITLE
about loss D_real or D_real_logits use ?

### ISSUE THREAD

**Issue by ranjiewwen** [hwalsuklee/tensorflow-generative-model-collections#28]
thanks you code , when i read the code , the discriminator use sigmoid output, but you also return D_real_logits ,and when caluate the loss real or fake ,you use D_real_logits or D_fake_logits before sigmoid output ; why you don't use the D_real or D_fake ,the sigmoid ouput? 
`        # get loss for discriminator
        d_loss_real = - tf.reduce_mean(D_real_logits)
        d_loss_fake = tf.reduce_mean(D_fake_logits)`

thank ,can you help me?
